sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/XXE/XmlReaderSettingsValidator.cs

```diff
@@ -28,39 +29,54 @@
 
 namespace SonarAnalyzer.Rules.XXE
 {
+    /// <summary>
+    /// This class is responsible to check if a XmlReaderSettings node is vulnerable to XXE attacks.
+    ///
+    /// By default the class is safe:
```

Comment:
```suggestion
    /// By default the XmlReaderSettings is safe:
```

Reply:
"This class" at line 33 has a different meaning than "the class" at line 35. Better be explicit.